__mocks__/rn-fetch-blob.js

```diff
@@ -9,11 +52,15 @@ export default {
     }
   },
   fs: {
+    readdirSync: readdirSync,
     exists: existsMock,
+    writeFile: writeMockFile,
+    readFile: readMockFile,
```

Comment:
```suggestion
    readFile:  jest.fn(readMockFile),
```

Add jest function mocks to all the exported functions here.